plover/main.py

```diff
@@ -48,14 +48,19 @@ def init_config_dir():
         os.makedirs(CONFIG_DIR)
 
     # Copy the default dictionary to the configuration directory.
-    if not os.path.exists(DEFAULT_DICTIONARY_FILE):
-        unified_dict = {}
-        dict_filenames = glob.glob(os.path.join(ASSETS_DIR, '*.json'))
-        for dict_filename in dict_filenames:
-            unified_dict.update(json.load(open(dict_filename, 'rb')))
-        ordered = OrderedDict(sorted(unified_dict.iteritems(), key=lambda x: x[1]))
-        outfile = open(DEFAULT_DICTIONARY_FILE, 'wb')
-        json.dump(ordered, outfile, indent=0, separators=(',', ': '))
+    def copy_dictionary_to_config(name):
+        source_path = os.path.join(ASSETS_DIR, name)
+        out_path = os.path.join(CONFIG_DIR, name)
+        if not os.path.exists(out_path):
```

Comment:
This all looks good. If you want to rename main.json, what about defaultdict.json or plovertheory.json or something along those lines?